MantidQt/CustomInterfaces/src/Muon/MuonAnalysis.cpp

```diff
@@ -1660,9 +1660,18 @@ void MuonAnalysis::plotSpectrum(const QString &wsName, bool logScale) {
   s << "      layer.removeCurve(i)"; // remove everything else
 
   // Plot data in the given window with given options
-  s << "def plot_data(ws_name, errors, connect, window_to_use):";
-  s << "  w = plotSpectrum(ws_name, 0, error_bars = errors, type = connect, "
-       "window = window_to_use)";
+  s << "def plot_data(ws_name,errors, connect, window_to_use):";
+  if (m_uiForm.frontPlotFuncs->currentText().toStdString() == "Asymmetry") {
+    s << "  w = plotSpectrum(source=ws_name, indices=0,  "
+         "distribution=mantidqtpython.MantidQt.DistributionFalse,error_bars = "
+         "errors, type = connect, "
+         "window = window_to_use)";
+  } else {
+    s << "  w = plotSpectrum(source=ws_name, indices=0,  "
+         "distribution=mantidqtpython.MantidQt.DistributionDefault,error_bars "
+         "= errors, type = connect, "
+         "window = window_to_use)";
+  }
```

Comment:
Could the formatting be tidied up a bit here so the Python is easier to read? Perhaps one parameter per line. Spaces around `=` should be consistent, parameter name and value should not go on two separate lines.

I am guessing Clang format may have messed up whatever you had first!